splitio/client/input_validator.py

```diff
@@ -517,3 +518,46 @@ def valid_properties(properties):
         _LOGGER.warning('Event has more than 300 properties. Some of them will be trimmed' +
                         ' when processed')
     return True, valid_properties if len(valid_properties) else None, size
+
+def validate_pluggable_adapter(config):
+    """
+    Check if pluggable adapter contains the expected method signature
+
+    :param config: config parameters
+    :type config: Dict
+
+    :return: True if no issue found otherwise False
+    :rtype: bool
+    """
+    if config.get('storageType') != 'PLUGGABLE':
```

Comment:
besides `storageType` and `wrapper`, should we also validate the `prefix` config option here?

Reply:
Sure, will add validation that it is string type.